dependencies.gradle

```diff
@@ -2,7 +2,7 @@ ext.versions = [
 
         supportLibrary : '27.1.0',
         androidKtx : '0.2',
-        seismic : '1.0.2',
+        seismic : '1.0.3',
```

Comment:
No breaking change from 1.0.2 to 1.0.3
https://github.com/square/seismic/blob/master/CHANGELOG.md#version-103-2021-09-28